BoldGrid/library
# Ensure correct library versions are set during bulk activation.

This branch was created back in Nov. Because it modifies files in src/Util, it was put on hold for a bit...

To test:
1. Pick 2 plugins that _include the library_ and make sure they're on the **dev** branch of the library.
1. Bulk activate the two plugins and check the library versions, they should be the same. _This is the error we're fixing._
1. Change both library versions to the **bulk-activation** branch.
1. Bulk activate the two plugins and check the library versions, they should be the appropriate versions.

This is a screenshot showing what the problem is:
![image](https://user-images.githubusercontent.com/5925563/57869390-f17f4c80-77d2-11e9-8ba7-eff69a12f2f7.png)
